clustering/kmeans_sample.ipynb

```diff
@@ -0,0 +1,518 @@
+{
```

Comment:
<p>There are too many things need to be refactored/improved, such as variable &amp; function naming, using int instead of float, and it's even not a typical K-means (no WCSS included). </p><p> </p><p>If you think this part is important enough to be kept in this file, I can send a PR to update this part.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Women-in-Data-Science-Taipei/2020-WiDS-Taipei-MLCC-Workshop/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>